packages/backend/migrations/20230918113102_rename_payment_pointer_tables.js

```diff
@@ -40,21 +40,27 @@ exports.up = function (knex) {
       table.dropIndex(['paymentPointerId', 'createdAt', 'id'])
       table.renameColumn('paymentPointerId', 'walletAddressId')
       table.foreign('walletAddressId').references('walletAddresses.id')
-      table.index(['walletAddressId', 'createdAt', 'id'])
+      table.index(['walletAddressId'])
+      table.index(['createdAt'])
+      table.index(['id'])
```

Comment:
I think these (and similar) should be re-added as a composite index instead? https://github.com/interledger/rafiki/pull/1937#discussion_r1357183892

Reply:
This hasn't been rebased off of the feature branch yet, so it doesn't have the latest changes I made to the db table migration. Should go away once I do that.